src/components/presentational/TimeSeriesChart/TimeSeriesChart.stories.tsx

```diff
@@ -96,7 +98,7 @@ async function getRandomIntradayData(start: Date) {
             timestamp: currentTime,
             value: (await predictableRandomNumber(currentTime.toISOString())) % 200
         });
-        currentTime = add(currentTime, { minutes: 5 });
+        currentTime = add(currentTime, { minutes: intervalMinutes ?? 5 });
         console.log(currentTime);
```

Comment:
Clean out console logs

Reply:
I went through the file and removed all the console logs.